superset/Dockerfile

```diff
@@ -95,6 +95,7 @@ RUN microdnf update \
 
 COPY superset/licenses /licenses
 COPY --from=builder /stackable ${HOME}
+COPY --from=statsd-exporter /bin/statsd_exporter /stackable/statsd_exporter
```

Comment:
Nit: When you already touch the Superset Image it would be great to include the fix from https://github.com/stackabletech/docker-images/pull/264/files to get rid of the `chown -R stackable:stackable /stackable`